src/DevBetterWeb.Web/Pages/User/UserBillingViewModel.cs

```diff
@@ -0,0 +1,53 @@
+﻿using DevBetterWeb.Core.Entities;
+using DevBetterWeb.Core.Interfaces;
+using System;
+using System.Collections.Generic;
+
+namespace DevBetterWeb.Web.Pages.User
+{
+  public class UserBillingViewModel
```

Comment:
ViewModel should be a DTO with no dependency injection, no behavior. Just properties and public get/set typically. Initialize properties inline to avoid null references.